partcad/src/partcad/runtime_python_conda.py

```
@@ -90,6 +90,29 @@ def once_conda_locked(self):
                     if not stderr is None and stderr.strip() != "":
                         pc_logging.error("conda env install error: %s" % stderr)
 
+                    # Install pip into the newly created conda environment
+                    p = subprocess.Popen(
```

Ruff (0.8.2)

`subprocess` call: check for execution of untrusted input

(S603)